src/ai/backend/manager/models/image.py

```diff
@@ -848,6 +851,7 @@ async def mutate(
                     sa.delete(ImageAliasRow).where(ImageAliasRow.image_id.in_(image_ids))
                 )
                 await session.execute(sa.delete(ImageRow).where(ImageRow.registry == registry))
+                await session.commit()
```

Comment:
Well... There's a `session.execute()` call here, so I guess we don't need to call additional `session.commit()`. Is it correct?

Reply:
my mistake. `session.commit()` is not needed here.